perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -353,6 +357,18 @@ sub inspect : Private {
 
         if ($valid) {
             $problem->update;
+            if ( defined($update_text) ) {
```

Comment:
This will be true if they have clicked save_inspected and not provided any update text.